trulens_eval/examples/quickstart/text2text_quickstart.ipynb

```diff
@@ -141,22 +151,70 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
    "metadata": {},
    "outputs": [],
    "source": [
     "from trulens_eval import TruBasicApp\n",
-    "tru_llm_standalone_recorder = TruBasicApp(llm_standalone, app_id=\"Happy Bot\", feedbacks=[f_answer_relevance])"
+    "\n",
+    "# Daniel - notice a regression behavior here, where only the recorder defined last (tru_llm_standalone_recorder_2) will be recording, anything above does not perform recording at all.\n",
```

Comment:
is this change intended? or just a temporary marker of a bug

Reply:
oops this was the minimal example I set up to repro the bug of TruBasicApp. Piotr is aware of this and I'll remove this change